website/docs/plugin-generator.md

```diff
@@ -19,6 +19,12 @@ then, install the custom generator running the following in your terminal.
 npm i -g generator-verdaccio-plugin
 ```
 
+:::caution
+
+generator-verdaccio-plugin does not compality yeoman version 5 now, you must use version 4 correctly.
```

Comment:
What exactly fails ?

Reply:
I've been debugging both the generator-verdaccio-plugin@4.1.0 and the latest version of generator-verdaccio-plugin locally. The issue I've encountered is that yeoman-api isn't compatible with yeoman-generator@5.7.0.

local yeoman-generate version:
![image](https://github.com/verdaccio/verdaccio/assets/42909374/8eae485d-fd6b-46eb-a2de-af03b3707891)

generator-verdaccio-plugin@4.1.0 yeoman-generate version:
![image](https://github.com/verdaccio/verdaccio/assets/42909374/fa63f0d8-c880-40b5-9962-96afacbe4394)

local version is fixed by verdaccio/generator-verdaccio-plugin#186
So, all you need to do is to publish the 'generator-verdaccio-plugin' based on the current content to fix this issue.🤣

Reply:
I will :) maybe I forgot doing it

Reply:
Yes, you can close this pr when the new plugin generator released.
Have a good time on your holiday!